File: Iniciante/1006/1006.c
#include <stdio.h>

int Media(double a, double b, double c, double *media){
    *media = (2*a+3*b+5*c)/10;
}

int main(){
    double a,b,c, media;
    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);
    Media(a,b,c, &media);
    printf("MEDIA = %.1lf\n", media);
    return 0;
}

File: Iniciante/1006/1006.cs
using System; 

class URI {

    static void Main(string[] args) { 
        double a, b, c, media;
        a = Convert.ToDouble(Console.ReadLine());
        b = Convert.ToDouble(Console.ReadLine());
        c = Convert.ToDouble(Console.ReadLine());
        media = (2*a+3*b+5*c)/10.0;
        Console.WriteLine($"MEDIA = {media:F1}");
    }

}